### PR TITLE
Added aedes vuln

### PR DESCRIPTION
**Pull request by mcollina**
Fixes https://github.com/nodejs/security-wg/issues/371

cc @lirantal

### CODE REVIEW

File: vuln/npm/457.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "id": 457,
+  "title": "Last Will message is not checked against authorization",
```
Comment (lirantal) [inline vuln/npm/457.json]: title should be the weakness type, i.e: `Cross-site Scripting (XSS) - Generic`
ref: https://cwe.mitre.org/

File: vuln/npm/457.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "overview": "Aedes does respect its own authorization rules when a client set a Last Will",
```
Comment (lirantal) [inline vuln/npm/457.json]: - does **not** ?
- s/set/sets ?

File: vuln/npm/457.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "cves": [
+  ],
+  "vulnerable_versions": "<0.35.0",
+  "patched_versions": ">=0.35.1",
+  "recommendation": "Update aedes module to version >= 0.35.1",
+  "references": [
+    "https://github.com/mcollina/aedes/issues/211",
+    "https://github.com/mcollina/aedes/issues/212"
+  ],
+  "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:C/C:N/I:L/A:N/E:H/RL:O/RC:C/CR:X/IR:X/AR:X/MAV:X/MAC:L/MPR:L/MUI:N/MS:C/MC:N/MI:L/MA:N",
```
Comment (lirantal) [inline vuln/npm/457.json]: we usually just score them with the base score and without environmental or other affects, so this should probably read: `CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:C/C:N/I:L/A:N` and that scores 5.0

**Comment by mcollina** [comment]
@lirantal done.

**Comment by lirantal** [comment]
Looks good, thanks!

**Pull request merged**